src/dsa/rpo_falcon512/mod.rs

```diff
@@ -42,16 +41,19 @@ pub const PK_LEN: usize = 897;
 pub const SK_LEN: usize = 1281;
 
 /// Signature length as a u8 vector.
-const SIG_LEN: usize = 626;
+const SIG_LEN: usize = 625;
 
 /// Bound on the squared-norm of the signature.
 const SIG_L2_BOUND: u64 = 34034726;
 
+/// Standard deviation of the Gaussian over the lattice.
+const SIGMA: f64 = 165.7366171829776;
+
 // TYPE ALIASES
 // ================================================================================================
 
-type SignatureBytes = [u8; NONCE_LEN + SIG_LEN];
+type SignatureBytes = [u8; SIG_HEADER_LEN + SIG_NONCE_LEN + SIG_LEN];
 type PublicKeyBytes = [u8; PK_LEN];
-type SecretKeyBytes = [u8; SK_LEN];
-type NonceBytes = [u8; NONCE_LEN];
+type NonceBytes = [u8; SIG_NONCE_LEN];
 type NonceElements = [Felt; NONCE_ELEMENTS];
+type B0 = [Polynomial<i16>; 4];
```

Comment:
Is there a way to rename `B0` into something more descriptive?

Reply:
I changed it to `ShortLatticeBasis`, let me know if this doesn't work